paper-input-behavior.html

```diff
@@ -264,10 +264,23 @@
     /**
      * Validates the input element and sets an error style if needed.
      */
-     validate: function () {
+     validate: function() {
        return this.inputElement.validate();
      },
 
+    /**
+     * Restores the cursor to its original position after updating the value.
+     */
+    updateValueAndPreserveCaret: function(newValue) {
+      var start = this.inputElement.selectionStart;
```

Comment:
I wonder if you need to check for these properties, since somebody can use a custom element as the `inputElement`?